dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/services/BulkDataRequestEmailBuilder.kt

```diff
@@ -52,7 +52,7 @@ class BulkDataRequestEmailBuilder(
                 "User" to buildUserInfo(DatalandAuthentication.fromContext() as DatalandJwtAuthentication),
                 "Reporting Periods" to bulkDataRequest.reportingPeriods.joinToString(", "),
                 "Requested Frameworks" to bulkDataRequest.dataTypes.joinToString(", "),
-                "Accepted Company Identifiers" to acceptedCompanyIdentifiers.joinToString(", "),
+                "Accepted Companies (Dataland IDs)" to acceptedDatalandCompanyIds.joinToString(", "),
```

Comment:
Maybe, can you provide also the company name like in the single data request internal mail?

Reply:
https://jira.d-fine.dev/browse/DALA-2812

Reply:
there is one

Reply:
After quick call: If there is an easy way to add the company name(s) to the email, this would be nice to have. Otherwise the story DALA-2812 addresses it

Reply:
Would be possible, would impact a few code files. A own small ticket is not too bad.

Reply:
okay, will leave it as it is. And let the DALA-2812 do its work.